ProjectTemplates/VisualStudio2010/iOS/Game1.csproj

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\MonoGame\v3.0\MonoGame.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\MonoGame\v3.0\MonoGame.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">iPhoneSimulator</Platform>
+    <ProjectTypeGuids>{FEACFBD2-3405-455C-9665-78FE426C6842};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <ProjectGuid>$guid1$</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>$safeprojectname$</RootNamespace>
+    <IPhoneResourcePrefix>Resources</IPhoneResourcePrefix>
+    <AssemblyName>$safeprojectname$</AssemblyName>
+    <MonoGamePlatform>iOS</MonoGamePlatform>
+    <MonoGameContentBuilderExe></MonoGameContentBuilderExe>
+    <MonoGameInstallDirectory Condition=" '$(OS)' != 'Unix' ">$(MSBuildProgramFiles32)</MonoGameInstallDirectory>
+    <MonoGameInstallDirectory Condition=" '$(OS)' == 'Unix' ">$(MSBuildExtensionsPath)</MonoGameInstallDirectory>    
```

Comment:
Weird ... the code for those lines was removed in
https://github.com/mono/MonoGame/commit/e80b65f27be8696f25c3f67f3937c60b40a0565d#diff-ef34073268fbfd9c847f53624108853bL44

And my local version didn't put them in..

On 29 February 2016 at 16:29, KonajuGames <notifications@github.com> wrote:

> In ProjectTemplates/VisualStudio2010/iOS/Game1.csproj
> <https://github.com/mono/MonoGame/pull/4582#discussion_r54435365>:
>
> > +<?xml version="1.0" encoding="utf-8"?>
> > +<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
> > +  <Import Project="$(MSBuildExtensionsPath)\MonoGame\v3.0\MonoGame.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\MonoGame\v3.0\MonoGame.Common.props')" />
> > +  <PropertyGroup>
> > +    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
> > +    <Platform Condition=" '$(Platform)' == '' ">iPhoneSimulator</Platform>
> > +    <ProjectTypeGuids>{FEACFBD2-3405-455C-9665-78FE426C6842};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
> > +    <ProjectGuid>$guid1$</ProjectGuid>
> > +    <OutputType>Exe</OutputType>
> > +    <RootNamespace>$safeprojectname$</RootNamespace>
> > +    <IPhoneResourcePrefix>Resources</IPhoneResourcePrefix>
> > +    <AssemblyName>$safeprojectname$</AssemblyName>
> > +    <MonoGamePlatform>iOS</MonoGamePlatform>
> > +    <MonoGameContentBuilderExe></MonoGameContentBuilderExe>
> > +    <MonoGameInstallDirectory Condition=" '$(OS)' != 'Unix' ">$(MSBuildProgramFiles32)</MonoGameInstallDirectory>
> > +    <MonoGameInstallDirectory Condition=" '$(OS)' == 'Unix' ">$(MSBuildExtensionsPath)</MonoGameInstallDirectory>
>
> It was the latest development MacOS X package from a few days ago (Feb 27).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mono/MonoGame/pull/4582/files#r54435365>.
>